lib/screens/home/home_screen.dart

```diff
@@ -14,46 +18,55 @@ class HomeScreen extends ConsumerStatefulWidget {
 
 class _HomeScreenState extends ConsumerState<HomeScreen> {
   int _selectedIndex = 0;
-  final List<TabScreen> _screens = [
-    for (final screen in TabScreen.values) screen,
-  ];
 
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      appBar: AppBar(
-        forceMaterialTransparency: true,
-        title: Text(
-          _screens[_selectedIndex].label,
-          style: const TextStyle(fontWeight: FontWeight.w600),
-        ),
-        actions: [
-          if (_selectedIndex == 4)
-            IconButton(
-              icon: const Icon(Icons.rotate_left),
-              onPressed: () {
-                ref.read(geminiMessagesProvider.notifier).clearChatHistory();
-              },
+    return Consumer(
+      builder: (context, ref, _) {
+        final screens = TabScreen.getScreens(ref);
+        final selectedLanguage = ref.watch(languageProvider);
+        final strings = selectedLanguage == LanguageSettings.defaultLanguage
+            ? StringsEn.en
+            : StringsPl.pl;
+
+        return Scaffold(
+          appBar: AppBar(
+            forceMaterialTransparency: true,
+            title: Text(
+              screens[_selectedIndex].label,
+              style: const TextStyle(fontWeight: FontWeight.w600),
```

Comment:
No need for consumer in ConsumerWidget or ConsumerStatefulWidget, fix in all files